docs/UserGuide.md

```diff
@@ -132,6 +136,130 @@ Examples:
 * `addLocation idl/L123 n/Vivocity a/John street, block 123, #01-01`
 * `addLocation idl/L234 n/Betsy Crowe's House a/Newgate Prison`
 
+#### Adding data from CSV files
+
+As you may have pre-existing data stored in the Excel file format, VirusTracker provides a way to import data directly from
+files in the CSV format. Excel provides an option to save existing _.xlsx_ extension files as _.csv_ files.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Note:**
+If you are importing data from a pre-existing Excel file, you may have to first format it to a format that is readable by VirusTracker.
+</div>
+
+You may read more about it [here](#format-for-csv-files).
+
+Format: `addFromCsv FILE_PATH l/LIST_TYPE`
+
+* `FILE_PATH` refers to the absolute file path where the CSV file resides.
+  * For example, if you wish to import data from `personList.csv` located in your desktop, the absolute file path could look
+  something like this: `C:/Users/user/Desktop/personList.csv` _(for Windows)_
+  * You may **find the absolute file path** as follows
+    1. Right-click your file `E.g. personList.csv`
+    2. Select 'Properties'
+    3. Take note of the path specified in the 'Location' field. `E.g. C:/Users/user/Desktop`
+    4. The absolute file path is the path found in Step 3 along with your file name. `C:/Users/user/Desktop/personList.csv`
+* `LIST_TYPE` refers to 'people', 'locations' or 'visits'.
+  * The prefix `l/` is also used for [listing data](#listing-data-list)
+* The CSV file should have its data in [VirusTracker readable format](#format-for-csv-files).
+* If you do not specify an absolute path, VirusTracker **may not be able to find your file!**
+
+Examples:
+* `addFromCsv C:/Users/alice/Desktop/peopleToAdd.csv l/people`
+* `addFromCsv D:/visits on Dec 20.csv l/visits`
+
+#### Exporting data to CSV files
+
+Often, you may not only work on a single device. 
+
+VirusTracker enables you to export the current data stored into a CSV file 
+which could then be read by the VirusTracker application on another device.
+
+Format: `exportToCsv FILE_PATH l/LIST_TYPE`
+
+* `FILE_PATH` refers to the absolute file path where the CSV file should reside.
+  * Refer to the [Adding data from CSV files](#adding-data-from-csv-files) section to find out the absolute path of a file.
+  * If the CSV file does not exist at the specified location, VirusTracker will create it for you.
+* `LIST_TYPE` refers to 'people', 'locations' or 'visits'.
+  * The prefix `l/` is also used for [listing data](#listing-data-list)
+* The CSV file will have its data in [VirusTracker readable format](#format-for-csv-files).
+* If you do not specify an absolute path, the file may be **created at an unexpected place!**
+
+Examples:
+* `exportToCsv C:/Users/alice/Desktop/peopleToAdd.csv l/people` creates a people data CSV file named `peopleToAdd.csv`
+* `addFromCsv D:/visits on Dec 20.csv l/visits` creates a visit data CSV file named `visits on Dec 20.csv`
```

Comment:
I think this may be a typo error? The use of "addFromCsv" is unexpected.

Reply:
Done